project/Http4sPlugin.scala

```diff
@@ -37,22 +35,15 @@ object Http4sPlugin extends AutoPlugin {
     // Many steps only run on one build. We distinguish the primary build from
     // secondary builds by the Travis build number.
     isTravisBuild := sys.env.get("TRAVIS").isDefined,
-    http4sPrimary := sys.env.get("TRAVIS_JOB_NUMBER").fold(true)(_.endsWith(".1")),
```

Comment:
This has been ever since we introduced stages, and why we've had to manually tag and bump the last few releases.  It's obsolete.